ting-bill-split.go

```diff
@@ -317,6 +317,51 @@ func parseMegabytes(megReader io.Reader) (map[string]int, error) {
 	return m, nil
 }
 
+func createNewBillingDir(args []string) {
+	newDirName := "new-billing-period"
+	if len(args) > 2 {
+		fmt.Println("Syntax: `new <dir-name>`")
+	} else {
+		if len(args) == 2 {
+			newDirName = args[1]
+		}
+		if _, err := os.Stat(newDirName); os.IsNotExist(err) {
+			fmt.Println("Creating a directory for a new billing period.")
+			os.MkdirAll(newDirName, os.ModeDir)
```

Comment:
This makes a dir with no permissions so I can't write to it. Change `os.ModeDir` to `0755` for reasonable settings. Also check the error it returns.